elements/bulbs-slideshow/bulbs-slideshow.js

```diff
@@ -103,9 +109,18 @@ class BulbsSlideshow extends BulbsHTMLElement {
 
   afterSlideTransition () {
     this.render();
+    this.sendPageView.bind(this).call();
     this.navigateToNextSlide();
   }
 
+  sendPageView () {
+    this.analyticsManager.trackPageView(
```

Comment:
this would face the same issue as above - either binding context or passing in 4 values